bin/prx-dovetail-regions.rb

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby
+# View/update which Dovetail Router regions DNS is pointing at.
+
+require "bundler/inline"
+
+gemfile do
+  source "https://rubygems.org"
+  gem "awesome_print"
+  gem "aws-sdk-route53"
+  gem "prx-ruby-aws-creds"
+  gem "slop"
+end
+
+OPTS = Slop.parse do |o|
+  o.string "-p", "--profile", "AWS profile", default: "prx-legacy-route53"
+  o.string "-e", "--environment", 'E.g. "production" or "staging"', required: true
+  o.string "-c", "--change", 'Change dovetail regions (e.g., "us-east-1" or "all")', default: nil
```

Comment:
`change` threw me off at first, because I thought it was making some change _to_ the provided region itself, rather than _changing to_ the provided region.

Maybe `-s`/`--set-region` would be more self explanatory?

Reply:
I was avoiding "region" since it gets a bit conflated with the aws/sdk/cli `--region` param?  And oddly, the command name is `change_resource_record_sets` ... not "update" or "set".

But I'm not attached to the param name, if you think `--set-regions` would be clearer.